docs/.vitepress/theme/components/ArticleMetadata.vue

```diff
@@ -0,0 +1,74 @@
+<script lang="ts" setup>
+import { useData } from 'vitepress'
+import { computed, ref, onMounted } from 'vue'
+import { countWord } from '../untils/functions'
+
+const { page } = useData()
+const date = computed(
+  () => new Date(page.value.lastUpdated!)
+)
+
+const wordCount = ref(0)
+const imageCount = ref(0)
+
+const wordTime = computed(() => {
+    return ((wordCount.value / 275) * 60)
+})
+
+const imageTime = computed(() => {
+    const n = imageCount.value
+    if (imageCount.value <= 10) {
+        // 等差数列求和
+        return n * 13 + (n * (n - 1)) / 2
+    }
+    return 175 + (n - 10) * 3
+})
+
+// 阅读时间
+const readTime = computed(() => {
+    return Math.ceil((wordTime.value + imageTime.value) / 60)
+})
+
+
+function analyze() {
+    document.querySelectorAll('.meta-des').forEach(v => v.remove())
```

Comment:
在 `docs/.vitepress/theme/components/ArticleMetadata.vue` 中，`analyze` 函数直接操作 DOM (`document.querySelectorAll`)。虽然在 VitePress 环境中这通常是安全的，但最好确保组件在客户端渲染时才执行此操作，以避免 SSR 问题。

```suggestion
function analyze() {
  if (typeof window !== 'undefined') {
    document.querySelectorAll('.meta-des').forEach(v => v.remove())
    const docDomContainer = window.document.querySelector('#VPContent')
    const imgs = docDomContainer?.querySelectorAll('.content-container .main img')
    imageCount.value = imgs?.length || 0
    const words = docDomContainer?.querySelector('.content-container .main')?.textContent || ''
    wordCount.value = countWord(words)
  }
}